app/views/posts/_post.html.erb

```diff
@@ -0,0 +1,12 @@
+<article class="notification my-2">
+  <div class="mb-2">
+    <p><%= t('post', id: post.id)%></p>
+    <p class="is-size-7 is-italic"><%= t('posted_on', date: post.created_at.to_s(:nice_date)) %></p>
+  </div>
+  <%= link_to user_post_path(post.author.id, post.id) do %>
+    <h3 class="title"><%= post.title %></h3>
+  <% end %>
```

Comment:
- [ ] Post show page should display the post with all comments listed as shown on the wireframe. The following error occurs when a post is clicked. I recommend that you fix that.
![image](https://user-images.githubusercontent.com/29071984/197722850-50097331-8ce8-4ece-b338-b2d937a51856.png)